### DISCUSSION TITLE
Thumbnails are now evaluating to HTML img elements rather than URLs

### DISCUSSION THREAD

**Discussion by ChaDonSom** [lunarphp/lunar#637] [Q&A]
Hey everyone! 👋 

I just finished migrating from GetCandy to Lunar to see how things are going, and I noticed my thumbnails all being broken.

It appears that the `purchasable->getThumbnail()` method now returns something like

```html
<img
  src="http://localhost:8000/storage/products/2022/10/12/YLLOWZbBcOULYpxpuHVeFXxQU4cZLX-metaUFhMXzIwMjIwODEwXzE3MTUzMDQ2My5qcGc=-.jpg"
  alt="YLLOWZbBcOULYpxpuHVeFXxQU4cZLX-metaUFhMXzIwMjIwODEwXzE3MTUzMDQ2My5qcGc=-"
>
```

but it used to return just the URL, i.e.

```
http://localhost:8000/storage/products/2022/10/12/YLLOWZbBcOULYpxpuHVeFXxQU4cZLX-metaUFhMXzIwMjIwODEwXzE3MTUzMDQ2My5qcGc=-.jpg
```

I can update my storefront easily enough, but I wonder how to update the hub.

Looking around a little bit, I see this is a method from `spatie/laravel-medialibrary`. I wonder, did I update it to a new version that broke things? Or, what else might be going on?

Thank you for your time and understanding!

**Comment by glennjacobs** [comment]
Can you see what version of the media library package is installed? As you say, this functionality isn't provided by Lunar itself. 

@alecritson might have some more insight.

**Comment by alecritson** [comment]
This is expected behaviour of the media package.

There was a change in Lunar for the way a variant evaluates it's thumbnail, there might be something that needs updating on the Demo store repo still. If there are still points in the hub this is happening it would be good to know so that it can be fixed.

**Comment by ChaDonSom** [comment]
Hey guys, thank you for the quick reply!

In lines `171-172` of `resources/views/livewire/dashboard.blade.php` (at least in my `vendor` directory), there's a use of a `$thumbnail` as an `<img src>`, which makes the 'Top Selling Products' section display wrong.

**Reply by ChaDonSom** [reply]
I forgot to mention, this is in the hub.
